modules/oadp-restic-issues.adoc

```diff
@@ -0,0 +1,111 @@
+
+. Create a supplemental group for `Restic` on the NFS data volume.
+. Set the `setgid` bit on the NFS directories so that group ownership is inherited.
+. Add the `spec.configuration.restic.supplementalGroups` parameter and the group ID to the `DataProtectionApplication` manifest, as in the following example:
++
+[source,yaml]
+----
+spec:
+  configuration:
+    restic:
+      enable: true
+      supplementalGroups:
+      - <group_id> <1>
+----
+<1> Specify the supplemental group ID.
+
+. Wait for the `Restic` pods to restart so that the changes are applied.
+
+[id="restic-restore-deploymentconfig-issue_{context}"]
+== Restore CR of Restic backup remains in "InProgress"
+
+The status of the `Restore` CR of a Restic backup remains in the `InProgress` phase.
+
+The `Velero` pod log displays the following error messages:
+
+* `Backup storage location is invalid, marking as unavailable`
+* `Current backup storage locations available/unavailable/unknown`
+
+.Cause
+
+The `DeploymentConfig` object redeploys the `Restore` pod, causing the `Restore` CR to fail.
+
+.Solution
+
+. Create a `Restore` CR that excludes the `ReplicationController` and `DeploymentConfig` resources:
++
+[source,terminal]
+----
+$ velero restore create --from-backup=<backup> -n openshift-adp \ <1>
+  --include-namespaces <namespace> \ <2>
+  --exclude-resources replicationcontroller,deploymentconfig \
+  --restore-volumes=true
+----
+<1> Specify the name of the `Backup` CR.
+<2> Specify the `include-namespaces` in the `Backup` CR.
+
+. Verify that the status of the `Restore` CR is `Completed`:
++
+[source,terminal]
+----
+$ oc get restore -n openshift-adp <restore> -o jsonpath='{.status.phase}'
+----
+
+. Create a `Restore` CR that includes the `ReplicationController` and `DeploymentConfig` resources:
++
+[source,terminal]
+----
+$ velero restore create --from-backup=<backup> -n openshift-adp \
+  --include-namespaces <namespace> \
+  --include-resources replicationcontroller,deploymentconfig \
+  --restore-volumes=true
+----
+
+. Verify that the status of the `Restore` CR is `Completed`:
++
+[source,terminal]
+----
+$ oc get restore -n openshift-adp <restore> -o jsonpath='{.status.phase}'
+----
+
+. Verify that the backup resources have been restored:
++
+[source,terminal]
+----
+$ oc get all -n <namespace>
+----
+
+[id="restic-backup-cannot-be-recreated-after-s3-bucket-emptied_{context}"]
+== Restic Backup CR cannot be recreated after bucket is emptied
+
+If you create a Restic `Backup` CR for a namespace, empty the S3 bucket, and then recreate the `Backup` CR for the same namespace, the recreated `Backup` CR fails.
+
+The `velero` pod log displays the error message, `msg="Error checking repository for stale locks"`.
+
+.Cause
+
+Velero does not create the Restic repository from the `ResticRepository` manifest if the Restic directories are deleted on S3 object storage. See (link:https://github.com/vmware-tanzu/velero/issues/4421[Velero issue 4421]) for details.
```

Comment:
Could you please change the "S3 object storage" to just "object storage"? it applies to all kinds of object storages.